sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -573,6 +604,26 @@ joint_matrix_mad(
 #endif // defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
 }
 
+float float_to_tf32(float a) {
```

Comment:
This is in sync with the fact that an element indexing of joint matrix of type tf32 is of type float. 
joint_matrix<precision::tf32, TM, TK> sub_a(sg);
sub_a.get_wi_data()[i] = float_to_tf32(sub_a.get_wi_data()[i]);
sub_a.get_wi_data()[i] is of type float but numerically it is tf32 after this conversion. 
Please add this clarification as a comment.